gui/src/components/WithLeftWorkspaceNav.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+  import * as router from 'svelte-spa-router';
+
+  import Home from './mono/home.svelte';
+  import Layers from './mono/layers.svelte';
+  import Storage from './mono/database.svelte';
+  import Networking from './mono/networking.svelte';
+
+  export let workspaceId: string;
+  export let active: string;
```

Comment:
I would suggest driving the `active` prop by whether the URL matches the link rather than passing in the active link as a prop. I'm guessing that this is why you made it a higher-order component.

Reply:
I see what you mean, but I think this causes issues when you want to show the active menu item for a nested page (i.e. the process details page, which is accessed through the main dashboard using the "home" icon, should still highlight the home icon as the "active tab/page")

Reply:
We should use this: https://github.com/ItalyPaleAle/svelte-spa-router#highlight-active-links